flocker/node/agents/functional/test_cinder.py

```diff
@@ -49,7 +49,33 @@
 
 # Tests requiring virsh can currently only be run on a devstack installation
 # that is not within our CI system. This will be addressed with FLOC-2972.
-require_virsh = skipIf(
+#
+# In the meantime, you can do the following (provided you have access):
```

Comment:
Not done.

What's the reason for your preference?

I put it in the comments because:
 * it's temporary documentation until FLOC-2972 (mentioned in the comment) is fixed
 * no one outside ClusterHQ has access to the machine
 * the only time when people will want to follow these instructions is when they are doing stuff related to this code